<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dero Testnet Five Card Draw</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/images/pokerLogo.png</normaloff>:/images/pokerLogo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-image: url(:/images/background.png);	
}

QGroupBox {
    border: 2px solid gray;
    border-radius: 5px;
}

QPushButton:enabled{
	background-color: rgb(85, 88, 93, 90);
    border: 2px solid #8f8f91;
    border-radius: 6px;
    min-width: 80px;
}

QPushButton:hover {
	background-color: rgb(79, 186, 196, 150);
}

QPushButton:pressed {
	background-color: rgb(56, 47, 165);
}

QPushButton:!enabled
{
    border: 2px solid;
	border-color: rgb(143, 143, 145, 90);
    border-radius: 6px;
	color: rgb(0, 0, 0, 0);
    border-radius: 6px;
    min-width: 80px;
}

QDoubleSpinBox{
	background-color: rgb(0, 0, 0, 0);
    border-style: inset;
    border-width: 1px;
    border-radius: 6px;
    border-color: white;
    padding-right: 10px;
}

QDoubleSpinBox:hover {
	background-color: rgb(79, 186, 196, 30);
}
QDoubleSpinBox::up-button:pressed {
	background-color: rgb(56, 47, 165);
}

QDoubleSpinBox::up-button {
     width: 30px;
     subcontrol-origin: center;
     subcontrol-position: top right;
}

QDoubleSpinBox::down-button:pressed {
	background-color: rgb(56, 47, 165);
}

QDoubleSpinBox::down-button {
	width: 30px;
     subcontrol-origin: center;
     subcontrol-position: bottom right;
}

QDoubleSpinBox:!enabled{
	background-color: rgb(0, 0, 0, 0);
    border-style: inset;
    border-width: 1px;
    border-radius: 6px;
   border-color: rgb(143, 143, 145, 90);
    padding-right: 10px;
}

QComboBox::enabled {
	background-color: rgb(85, 88, 93, 90);
    border: 2px solid #8f8f91;
    border-radius: 6px;
    min-width: 80px;
}

QComboBox:!enabled
{
    border: 2px solid;
	border-color: rgb(143, 143, 145, 90);
    border-radius: 6px;
	color: rgb(0, 0, 0);
    border-radius: 6px;
    min-width: 80px;
}

QComboBox::drop-down {
	 width: 21px;
     subcontrol-origin: center;
     subcontrol-position: bottom right;
}

QComboBox:hover {
	background-color: rgb(79, 186, 196, 150);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="10" column="0" colspan="2">
     <widget class="QLineEdit" name="userInput">
      <property name="minimumSize">
       <size>
        <width>189</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>User:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="shuffleButton">
      <property name="text">
       <string>Shuffle Deck</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="5" colspan="5">
     <widget class="QTextBrowser" name="logTextBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 88, 93, 90);</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QCheckBox" name="p4CheckBox">
      <property name="text">
       <string>Player 4 Folded</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QCheckBox" name="p1CheckBox">
      <property name="text">
       <string>Player 1 Folded</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QComboBox" name="playersComboBox">
      <item>
       <property name="text">
        <string>Players</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Close Table</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QLineEdit" name="daemonRPCinput">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Enter Daemon RPC Address</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QComboBox" name="winnerComboBox">
      <item>
       <property name="text">
        <string>owner:</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player5</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QPushButton" name="daemonRPCbutton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QDoubleSpinBox" name="anteSpinBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="maximum">
       <double>1200.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QCheckBox" name="p3CheckBox">
      <property name="text">
       <string>Player 3 Folded</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QCheckBox" name="walletConnectedBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Connected</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="payoutPushButton">
      <property name="text">
       <string>Pay Winner</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QCheckBox" name="daemonConnectedBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Connected</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="ownerPushButton">
      <property name="text">
       <string>Owner Key</string>
      </property>
     </widget>
    </item>
    <item row="9" column="6">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255, 45);</string>
      </property>
      <property name="text">
       <string>♣♣♣♣♣♣
Table v0.1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QPushButton" name="entryPushButton">
      <property name="text">
       <string>Sit Down</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="6">
     <widget class="QGroupBox" name="groupBoxP1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="3">
        <widget class="QDoubleSpinBox" name="betSpinBox">
         <property name="maximum">
          <double>9999999999999.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="4">
        <widget class="QLabel" name="localPlayerCard2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>273</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>120</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="img.qrc">:/images/card53.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="4">
        <widget class="QLabel" name="localPlayerCard5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>273</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>120</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="img.qrc">:/images/card53.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QPushButton" name="dealHandPushButton">
         <property name="text">
          <string>Deal Hand</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="checkButton">
         <property name="text">
          <string>Check</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="drawPushButton">
         <property name="text">
          <string>Draw</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="4">
        <widget class="QLabel" name="localPlayerCard4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>273</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>120</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="img.qrc">:/images/card53.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QComboBox" name="drawComboBox">
         <property name="maxVisibleItems">
          <number>15</number>
         </property>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Card 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Card 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Card 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Card 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Card 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cards 1 &amp; 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cards 1 &amp; 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cards 1 &amp; 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cards 1 &amp; 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cards 2 &amp; 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cards 2 &amp; 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cards 2 &amp; 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cards 3 &amp; 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cards 3 &amp; 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cards 4 &amp; 5</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0" rowspan="4">
        <widget class="QLabel" name="localPlayerCard1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>273</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>120</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="img.qrc">:/images/card53.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="4">
        <widget class="QLabel" name="localPlayerCard3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>273</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>120</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="img.qrc">:/images/card53.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="betButton">
         <property name="text">
          <string>Bet</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QDoubleSpinBox" name="playerId">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="prefix">
          <string>Player ID: </string>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QDoubleSpinBox" name="dsbTurn">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="prefix">
          <string>Turn: </string>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0" colspan="5">
     <widget class="QTextBrowser" name="turnReadOut">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>51</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 88, 93, 90);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:21pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="overwriteMode">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QPushButton" name="helpButton">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QDoubleSpinBox" name="dsbSeats">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="prefix">
       <string>Seats: </string>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QLineEdit" name="walletRPCinput">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Enter Wallet RPC Address</string>
      </property>
     </widget>
    </item>
    <item row="10" column="5" rowspan="2" colspan="2">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/pokerLogo.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QCheckBox" name="p2CheckBox">
      <property name="text">
       <string>Player 2 Folded</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QDoubleSpinBox" name="anteIsDSB">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="prefix">
       <string>Ante is: </string>
      </property>
      <property name="maximum">
       <double>1200.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QPushButton" name="gameKeyPushButton">
      <property name="text">
       <string>Get ID</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QCheckBox" name="p5CheckBox">
      <property name="text">
       <string>Player 5 Folded</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="4">
     <widget class="QLineEdit" name="gameKeyLineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>750</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Game Key: </string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QLineEdit" name="passwordInput">
      <property name="minimumSize">
       <size>
        <width>189</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QPushButton" name="walletRPCbutton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QDoubleSpinBox" name="dsbPot">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="prefix">
       <string>Dero Pot: </string>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="maximum">
       <double>1000000000000000000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="6" rowspan="2">
     <widget class="QTextBrowser" name="txLogTextBrowser">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 88, 93, 90);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1254</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
